nmagent/requests.go

```diff
@@ -448,7 +448,7 @@ func (NCVersionListRequest) Method() string {
 
 // Path returns the path required to issue the request.
 func (NCVersionListRequest) Path() string {
-	return "/NetworkManagement/interfaces/api-version/1"
```

Comment:
Would there be any reason for a client to use the version 1 API going forward? Can't remember if we discussed this...

Reply:
Actually, I didn't find any official support for "/NetworkManagement/interfaces/api-version/1", so it's not correct here.
I only find support for "/NetworkManagement/interfaces/api-version/2", version 2 is an api without token.
or "NetworkManagement/interfaces/NetworkID/networkContainers/NCID/version/authenticationToken/%s/api-version/1" version 1 is an api with token.
But, I tested on machine, "/NetworkManagement/interfaces/api-version/1" returning the same result as "/NetworkManagement/interfaces/api-version/2"
However, I think we still should make it correct.

Reply:
@csfmomo Okay cool, makes sense to me then :)